data/styles/default/include/priorities_4_overlays.prio.txt

```diff
@@ -775,6 +775,7 @@ shop-butcher                                        # icon z16- (also has captio
 shop-cheese                                         # icon z16- (also has caption(optional) z16-)
 shop-chocolate                                      # icon z16- (also has caption(optional) z16-)
 shop-confectionery                                  # icon z16- (also has caption(optional) z16-)
+shop-pasta                                          # icon z18- and caption z16-17 (also has caption(optional) z18-)
```

Comment:
Keep intentionally, because after adding a unique icon, it will be from z16 like others.

Reply:
It could be re-added when/if a unique icon is added.
As it is now it'll look very ugly and counter-intuitive with a caption on z16 changing to an icon when zoomed in.

Also actually might make sense to make it less visible than z16 because its a niche/specialized shop (like e.g. shop-coffee which is z18-).

Reply:
And actually what's the issue with a generic icon on z16?

Its much better than inconsistency with lone captions (for context: https://github.com/organicmaps/organicmaps/issues/4854)

Reply:
> Also actually might make sense to make it less visible than z16 because its a niche/specialized shop (like e.g. shop-coffee which is z18-).

I suppose that it's niche characteristic is depending on culture.
It's a shop selling food, around here, nowadays, most of them sells the pasta that you cook in your house and also sells cooked pasta ready to eat (takeaway). Some ot them also have a couple of seats to also eat there.